spec/controllers/courses_controller_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'spec_helper'
+
+RSpec.describe CoursesController, type: :controller do
+  let(:user) { create :user }
+
+  before do
+    sign_in user
+  end
```

Comment:
тут лучше сделать как инлайн блок, когда у тебя всего одна строка
```suggestion
  before { sign_in user }
```
исправление по желанию

Reply:
исправил